roles/ceph-infra/tasks/setup_ntp.yml

```diff
@@ -45,7 +45,7 @@
 
     - name: enable chronyd
       service:
-        name: chronyd
+        name: chrony
```

Comment:
chronyd is the daemon name on RedHat/CentOS so this change will break it.
So this would require some conditional between linux distribution.